tests/run-test.py

```
@@ -335,35 +374,45 @@ def parse_exe_match(path, executor, executor_tidb, executor_func, executor_curl_
+                    break
+                if line.startswith(TODO_PREFIX):
+                    todos.append(line[len(TODO_PREFIX):].strip())
+                    continue
+                if line.startswith(COMMENT_PREFIX) or len(line) == 0:
+                    continue
+                if origin.startswith(UNFINISHED_1_PREFIX) or origin.startswith(UNFINISHED_2_PREFIX):
+                    if cached[-1] == ',':
+                        cached += ' '
+                    cached += line
+                    continue
+                if cached != None and not matcher.on_line(cached, line_number_cached):
```

Ruff (0.15.14)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)